cpp/examples/1billion/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.
+
+cmake_minimum_required(VERSION 3.26.4)
+
+include(../set_cuda_architecture.cmake)
+
+# initialize cuda architecture
+rapids_cuda_init_architectures(billion)
+rapids_cuda_set_architectures(RAPIDS)
+
+project(
+  billion
```

Comment:
Can we rename this to `brc`? Or does the project need a unique name?

It's a little confusing to have three unique names for this example:
1. The directory is named `1billion`
2. The project is named `billion`
3. The executable is named `brc` (and variations thereof)

Let's consolidate these.

Reply:
I agree with renaming the directory and project the same. I want to use something easily readable/discoverable from the perspective of someone looking at the examples folder. My suggestion is to use `billion_rows` for the directory and project name. 
I would like to keep the executable names shorter since they are built in context of the parent directory and would look less cumbersome in my opinion. I'd like to keep the `brc` variations also because the blog uses those names in charts that would need to be regenerated.

Reply:
That’s fine! Let’s do that.